### DISCUSSION TITLE
Welcome to fcitx5 Discussions!

### DISCUSSION THREAD

**Discussion by wengxt** [fcitx/fcitx5#276] [General]
We'd like to use this to answer question, have disucussion and share ideas.

"Issues" is already used by lots of people to ask question and the google group (It's still gonna be used) does not seems to be a popular option to most of people.

Free free to use this place for question about configuration, setup, or customization.

(Due to the fact that most developer are Chinese, it's ok to use Chinese for post).

**Comment by englianhu** [comment]
[Ask a Favor : KDE Plasma 5 Input Methods](https://d.cosx.org/d/422243-ask-a-favor-kde-plasma-5-input-methods)

**Comment by xingwozhonghua126** [comment]
wengxt yyds
